whois-query/src/main/java/net/ripe/db/whois/query/query/Query.java

```diff
@@ -694,6 +700,10 @@ public String getCharsetName(){
         }
     }
 
+    public OverrideCredential getOverride() {
```

Comment:
may be use auto-generated getter and setter method names? Or is it like this for rest of getter methods too ?

Reply:
I used auto-generated but the name of the attribute is `override` so this is why you see getOverride
I will rename it to `overrideCredential` so it will change to getOverrideCredential